.github/workflows/macos.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         cmake
         cpprestsdk
         dylibbundler
-        ffmpeg@4
+        ffmpeg@6

```

Comment:
Can't we just remove this entirely as we're building from ffmpeg soure?

Reply:
ffmpeg@6 is pulled in by dependencies ...

But I think someone (@ooshlablu ;-) ) takes the changes and integrate the needed into his pr.

Reply:
@Baklap4 so yes, we can remove the entry :)